src/sentry/static/sentry/app/routes.jsx

```diff
@@ -1750,6 +1750,15 @@ function routes() {
               }
               component={errorHandler(LazyLoad)}
             />
+            <Route
+              path="/organizations/:orgId/performance/summary/rum"
```

Comment:
do we want the url to be `rum` because `realusermonitoring` is too long?

Reply:
realusermonitoring is way too long.

Reply:
I'm open to suggestions here, but yes the reason I picked rum over realusermonitoring was because it was too long.